modules/user-guide/pages/setting-up-the-bitbucket-application-link.adoc

```diff
@@ -0,0 +1,94 @@
+
+.Procedure
+
+. On a command line, generate the *privatepkcs8-stripped.pem* key in the required format:
++
+----
+$ openssl genrsa -out private.pem 2048
+
+$ openssl pkcs8 -topk8 -inform pem -outform pem -nocrypt -in private.pem -out privatepkcs8.pem
+
+$ cat privatepkcs8.pem | sed 's/-----BEGIN PRIVATE KEY-----//g' |  sed 's/-----END PRIVATE KEY-----//g' | tr -d '\n' > privatepkcs8-stripped.pem
+----
+
+. On a command line, generate the *public-stripped.pub* key in the required format:
++
+----
+$ openssl rsa -in private.pem -pubout > public.pub
+
+$ cat public.pub | sed 's/-----BEGIN PUBLIC KEY-----//g' | sed 's/-----END PUBLIC KEY-----//g' | tr -d '\n' > public-stripped.pub
+
+----
+
+. On a command line, generate a consumer key:
++
+----
+$ openssl rand -base64 24 > bitbucket-consumer-key
+----
+
+. On a command line, generate a shared secret:
++
+----
+$ openssl rand -base64 24 > bitbucket-shared-secret
+----
+
+. Go to *Administration* *>* *Application Links*
+//I can't find *Administration* when I am logged in to Bitbucket. max-cx
+//using https://youtu.be/djXwI9OGvhQ?t=117 as a reference (also asking the SME). max-cx
+//in the above YouTube tutorial: `<host>/bitbucket/plugins/servlet/applinks/listApplicationLinks`
+
+. Enter `{prod-url}/` into the URL field and click *Create new link*.
+
+. Under *The supplied Application URL has redirected once*, check the *Use this URL* checkbox and click *Continue*.
+
+. Enter *{prod-short}* as the *Application Name*.
+
+. Select *Generic Application* as the *Application Type*.
+
+. Enter *{prod-short}* as the *Service Provider Name*.
+
+. Paste the content of the *bitbucket-consumer-key* file as the *Consumer key*.
+
+. Paste the content of the *bitbucket-shared-secret* file as the *Shared secret*.
+
+. Enter `__<bitbucket_server_url>__/plugins/servlet/oauth/request-token` as the *Request Token URL*.
+
+. Enter `__<bitbucket_server_url>__/plugins/servlet/oauth/access-token` as the *Access token URL*.
+
+. Enter `__<bitbucket_server_url>__/plugins/servlet/oauth/access-token` as the *Authorize URL*.
+
+. Check the *Create incoming link* checkbox.
+
+. Click *Continue*.
+
+. Paste the content of the *bitbucket_consumer_key* file as the *Consumer Key*.
+
+. Enter *{prod-short}* as the *Consumer name*.
+
+. Paste the content of the *public-stripped.pub* file as the *Public Key*.
+
+. Click *Continue*.
+
+. Give the generated *privatepkcs8-stripped.pem* and *bitbucket_consumer_key* files to the administrator of the {prod-short} instance for your organization. The administrator will need these files when xref:administration-guide:applying-the-bitbucket-application-link-secret.adoc[applying the Bitbucket application link Secret].
+
+.Additional resources
+
+* link:https://confluence.atlassian.com/adminjiraserver/using-applinks-to-link-to-other-applications-938846918.html[Atlassian Documentation: Link to other applications]
```

Comment:
I think this link would be better since : https://confluence.atlassian.com/bitbucketserver/link-to-other-applications-1018764620.html since the current link in the PR seems to be docs related to Jira applications